src/translations/en-US.json

```diff
@@ -848,7 +848,7 @@
         "limit": 0
     },
     "mvf.treatNullValuesAsZeroTooltip": {
-        "value": "Blank values will be included if zero matches the filter condition.",
+        "value": "If the filter condition includes 0, blank values will be replaced with 0 and included in calculations.",
         "comment": "",
```

Comment:
We have to give translators some context: 
- how to understand that filter "includes 0"
- what is "blank" value
- how to understand "included in calculations"

Reply:
How about now?